### PR TITLE
[ICDS dashboard]: script to verify the data between ppd and par

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
##### SUMMARY
script to check whether data shown by the ppr as ppd is same as consistent

### CODE REVIEW

File: custom/icds_reports/reports/poshan_progress_dashboard_data.py [hunk 1]
```diff
@@ -31,13 +31,22 @@ def calculate_percentage_single_row(row, truncate_out=True):
     return row
 
 
-def calculate_aggregated_row(data, aggregation_level, data_format):
+def calculate_aggregated_row(data, aggregation_level, data_period, unique_id):
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/poshan_progress_dashboard_data.py:34]: I think following changes we have merged already to master. Why they are coming here again?

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
hey you will be limited releasing this yeah. So can you change the PR base from master to the latest release of TCL.

File: custom/icds_reports/reports/poshan_progress_dashboard_data.py (hunk 1 above)
Comment (ShubhamBansal1997) [reply custom/icds_reports/reports/poshan_progress_dashboard_data.py:34]: This branch is not based on master it's based on the setup release which I will deploy to TCL environment to test

**Review by Rohit25negi — APPROVED** [review]
looks good, Adding do not merge tag as this is to be limited release

**Comment by Rohit25negi** [comment]
Seeing the tag already there.

**Pull request closed without merging**